app/src/components/governance/ProposalStatusBar.vue

```diff
@@ -37,7 +37,10 @@
         bar-color="var(--highlight)"
       />
     </div>
-    <div class="bottom row">
+    <div
+      v-if="status.value !== governanceStatusEnum.DEPOSITING"
```

Comment:
```suggestion
      v-if="status.value === governanceStatusEnum.VOTING"
```

It only makes actually sense when it is on Voting Period

Reply:
don't we want to show the vote for passed and rejected proposals?

Reply:
As Fabo said, we cannot display those votes

Reply:
we can. we cannot show the percentages:
![Screen Shot 2020-09-21 at 10 35 48 AM](https://user-images.githubusercontent.com/6021933/93780481-38b2f400-fbf6-11ea-940b-5d9e2157c354.png)
.

Reply:
Ah, ok. Cool